tools/build_variables.py

```diff
@@ -171,8 +176,6 @@ def torch_vars():
             "-Icaffe2/torch/csrc",
             "-Icaffe2/torch/csrc/nn",
             "-Icaffe2/torch/lib",
-            "-DUSE_CPU_FUSER_FBCODE=1",
```

Comment:
Are we still disabling the CPU Fuser in fbcode? I'm not sure where that logic lives... (hopefully it is in a buck file somewhere)

Reply:
I copied the rules for what was there. According to this code it is enabled in fbcode.